tests/core/slice/test_core_slice.odin

```diff
@@ -180,3 +181,59 @@ test_sort_by_indices :: proc(t: ^testing.T) {
 		}
 	}
 }
+
+@test
+test_binary_search :: proc(t: ^testing.T) {
```

Comment:
The test specific `log` and `expect` procs defined at the top of the file should be used instead of `fmt.printf` and `assert`